src/App.jsx

```diff
@@ -1,5 +1,10 @@
 import 'bulma/css/bulma.css';
 import './App.scss';
+import { useState } from 'react';
+import cn from 'classnames';
+
+const SORT_BY_ALPHABETIC = 'alpabetic';
+const SORT_BY_LENGTH = 'length';
```

Comment:
tip:
it will be better to combine such kind sort types in common object
because  theoretically, the number of types can increase
It is also more convenient to keep constants in a separate file
```suggestion
export const SORT_TYPE = {
  ALPHABET: 'alphabet',
  LENGTH: 'length',
};
```